build/AppveyorBuildFunctions.ps1

```diff
@@ -3,10 +3,10 @@ function Build-Solution
     Write-Host "Building Event Hubs projects"
 
     dotnet restore  
```

Comment:
that should be `msbuild /t:restore` since there are some project files that `dotnet` can't parse...